lib/external/io.dart

```diff
@@ -23,11 +26,38 @@ ASTNode funcFree(Runtime runtime, ASTNode self, List<ASTNode> args) {
   final Scope scope = getScope(runtime, self);
 
   args.forEach((arg) {
-    if (arg is! VariableNode)
-      return;
+    if (arg is! VariableNode) return;
 
-    scope.variableDefinitions.removeWhere((varDef) => varDef.variableName == arg.variableName);
+    scope.variableDefinitions
+        .removeWhere((varDef) => varDef.variableName == arg.variableName);
   });
 
   return INITIALIZED_NOOP;
-}
\ No newline at end of file
+}
+
+Future<MapNode> funcSystem(
+    Runtime runtime, ASTNode self, List<ASTNode> args) async {
+  runtimeExpectArgs(
+      args, [ASTType.AST_STRING, ASTType.AST_LIST, ASTType.AST_BOOL]);
+
+  Map<String, dynamic> result = {
```

Comment:
It will be better if we get a return type of a certain class similar to ProcessResult instead of just a map

Reply:
Now it returns a <String, dynamic>